priv/perf/config/config.exs

```diff
@@ -45,4 +46,14 @@ config :logger, :console,
   discard_threshold: 2000,
   metadata: [:module, :function, :request_id, :trace_id, :span_id]
 
+config :statix,
+  host: "localhost",
+  port: 8125,
+  tags: [System.get_env("STATIX_TAG", "env:perf_test")]
```

Comment:
maybe it makes sense to yield error if this isn't set, otherwise we might overwrite data in datadog?